test_project/settings/base.py

```diff
@@ -161,7 +161,8 @@ def get_databases(base_dir):
 LANGUAGE_CODE = 'en-us'
 TIME_ZONE = 'UTC'
 USE_I18N = True
-USE_L10N = True
+if django.VERSION < (4, 0):
+    USE_L10N = True
```

Comment:
Fixes this warning in the test run:

```
../.tox/py310-dj42/lib/python3.10/site-packages/django/conf/__init__.py:267
  /Users/chainz/Documents/Projects/django-autocomplete-light/.tox/py310-dj42/lib/python3.10/site-packages/django/conf/__init__.py:267: RemovedInDjango50Warning: The USE_L10N setting is deprecated. Starting with Django 5.0, localized formatting of data will always be enabled. For example Django will display numbers and dates using the format of the current locale.
    warnings.warn(USE_L10N_DEPRECATED_MSG, RemovedInDjango50Warning)
```

As per https://docs.djangoproject.com/en/4.0/releases/4.0/#localization